fendermint/docker/builder.ci.Dockerfile

```diff
@@ -62,7 +62,7 @@ ARG TARGETARCH
 RUN if [ "${TARGETARCH}" = "arm64" ]; then \
   apt-get install -y g++-aarch64-linux-gnu libc6-dev-arm64-cross; \
   rustup target add aarch64-unknown-linux-gnu; \
-  rustup toolchain install ${RUST_VERSION}-aarch64-unknown-linux-gnu; \
+  rustup toolchain install ${RUST_VERSION}-aarch64-unknown-linux-gnu --force-non-host; \
```

Comment:
I think with latest rust you need to use `--force-non-host` for cross builds like this